### PR TITLE
Add id translation for create-cluster scenario

### PR DESCRIPTION
**Pull request by deryrahman**
This PR is part of Indonesian translation docs https://github.com/kubernetes/website/issues/22296 . It requires to translate Kubernetes first scenario in Katacoda labs ( this page `/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/`) 😊

cc: @girikuncoro

### CODE REVIEW

File: _id/1/step1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Kita telah memasang minikube untuk kami. Periksa apakah terpasang dengan baik, dengan menjalankan perintah *minikube version*:
```
Comment (girikuncoro) [inline _id/1/step1.md:1]: this sentence doesn't make sense, how about removing "untuk kami"?
```suggestion
Kita telah melakukan pemasangan minikube. Periksa apakah terpasang dengan baik, dengan menjalankan perintah *minikube version*:
```

File: _id/1/step1.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`minikube version`{{execute}}
+
+OK, kita dapat melihat minikube tersebut terpasang.
+
+Memulai klaster, dengan menjalankan perintah *minikube start*:
+
+`minikube start`{{execute}}
+
+Bagus! Kamu sekarang mempunyai klaster Kubernetes berjalan didalam terminal online kamu. Minikube memulai mesin virtual untuk kamu, dan klaster Kubernetes sekarang berjalan pada mesin virtual tersebut.
```
Comment (girikuncoro) [inline _id/1/step1.md:11]: nit: "di" needs space
```suggestion
Bagus! Kamu sekarang mempunyai klaster Kubernetes berjalan di dalam terminal online kamu. Minikube memulai mesin virtual untuk kamu, dan klaster Kubernetes sekarang berjalan pada mesin virtual tersebut.
```

File: _id/1/step2.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Untuk berinteraksi dengan Kubernetes selama bootcamp ini kita akan menggunakan antarmuka baris perintah, kubectl. Kita akan menjelaskan kubectl secara detail pada modul berikutnya, namun untuk sekarang, kita akan melihat informasi klaster.
```
Comment (girikuncoro) [inline _id/1/step2.md:1]:
```suggestion
Untuk berinteraksi dengan Kubernetes selama bootcamp ini kita akan menggunakan antarmuka baris perintah, kubectl. Kita akan menjelaskan kubectl secara detail pada modul berikutnya, namun untuk sekarang, kita akan melihat informasi klaster terlebih dahulu.
```

File: _id/1/step2.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+Untuk memeriksa apakah kubectl terpasang, kamu dapat menjalankan perintah *kubectl version*:
```
Comment (girikuncoro) [inline _id/1/step2.md:2]:
```suggestion
Untuk memeriksa apakah kubectl telah terpasang, kamu dapat menjalankan perintah *kubectl version*:
```

File: _id/1/step2.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+
+`kubectl version`{{execute}}
+
+OK, kubectl sudah dikonfigurasikan dan kita akan melihat versi klien dan juga server. Versi klien merupakan versi kubectl; versi server merupakan versi Kubernetes yang terinstall pada master. Kamu juga dapat melihat detail tentang _build_.
```
Comment (girikuncoro) [inline _id/1/step2.md:6]:
```suggestion
OK, kubectl sudah dikonfigurasikan dan kita akan melihat versi klien dan juga server. Versi klien merupakan versi kubectl; versi server merupakan versi Kubernetes yang telah terinstal pada master. Kamu juga dapat melihat detail tentang _build_.
```

File: _id/1/step3.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Mari melihat detail klaster. Kita akan melakukannya dengan menjalankan *kubectl cluster-info*:
+
+`kubectl cluster-info`{{execute}}
+
+Selama tutorial ini, kita akan berfokus kepada baris perintah untuk men-_deploy_ dan eksplorasi aplikasi kita.
```
Comment (girikuncoro) [inline _id/1/step3.md:5]: nit
```suggestion
Selama tutorial ini, kita akan fokus pada baris perintah untuk men-_deploy_ dan mengeksplorasi aplikasi kita.
```

File: _id/1/step3.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+Untuk melihat Node pada klaster, jalankan perintah *kubectl get nodes*:
+
+`kubectl get nodes`{{execute}}
+
+Perintah ini memperlihatkan semua Node yang dapat digunakan untuk hos aplikasi kita. Sekarang kita hanya mempunyai 1 Node, dan kita akan melihat bahwa statusnya siap (_ready_) (siap untuk menerima aplikasi sebagai _deployment_).
```
Comment (girikuncoro) [inline _id/1/step3.md:10]:
```suggestion
Perintah ini memperlihatkan semua Node yang dapat digunakan untuk hos aplikasi kita. Sekarang kita hanya mempunyai 1 Node, dan kita akan melihat bahwa statusnya telah siap (_ready_). Artinya, Node telah siap untuk menerima perintah untuk menjalankan aplikasi melalui _deployment_.
```

File: _id/1/intro.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+# Tujuan dari skenario interaktif ini adalah untuk men-_deploy_ pengembangan klaster Kubernetes lokal menggunakan minikube #
+
+Terminal online merupakan pra-konfigurasi lingkungan Linux yang dapat digunakan sebagai _console_ reguler (kamu bisa mengetik perintah).
+Klik pada `blocks of code` diikuti dengan sinyal ENTER akan mengeksekusi perintah tersebut dalam terminal.
```
Comment (girikuncoro) [inline _id/1/intro.md:4]:
```suggestion
Klik pada `blocks of code`, lalu tekan ENTER untuk mengeksekusi perintah tersebut di dalam terminal.
```

File: _id/1/index.json [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+	"details": {
+		"steps": [{
+			"title": "Klaster dimulai dan berjalan",
+			"text": "step1.md"
+		}, {
+			"title": "Versi klaster",
+			"text": "step2.md"
+		}, {
+			"title": "Detail klaster",
+			"text": "step3.md"
```
Comment (girikuncoro) [inline _id/1/index.json:23]: how about opening the steps with active verbs instead? seems more intuitive in Indonesian language for tutorial
```suggestion
			"title": "Memulai dan menjanlankan Klaster",
			"text": "step1.md"
		}, {
			"title": "Melihat versi klaster",
			"text": "step2.md"
		}, {
			"title": "Melihat detail klaster",
			"text": "step3.md"
```

**Comment by deryrahman** [comment]
> this is exciting, it's our first Kubernetes tutorial written in Indonesian language, thanks for doing this @deryrahman ! 🎉 I have few suggestions which can improve the instructions, let me know what you think

cool! thanks @girikuncoro, I'll make some changes based on your suggestions 👏

File: _id/1/step1.md (hunk 1 above)
Comment (deryrahman) [reply _id/1/step1.md:1]: oh ya, pardon, I missed this part 🙏

File: _id/1/index.json (hunk 9 above)
Comment (deryrahman) [reply _id/1/index.json:23]: Yup, more obvious

**Comment by BenHall** [comment]
@girikuncoro This looks great! We need to do something on our side to resolve an issue but this will be live tomorrow.

**Comment by BenHall** [comment]
@girikuncoro The content should now be available when embedding Katacoda. Please let us know if you need anything else

**Comment by girikuncoro** [comment]
Awesome, thanks a lot @BenHall for your help, we are planning to localize the remaining scenarios as well

**Pull request merged**